linux_os/guide/services/kerberos/package_krb5-server_removed/rule.yml

```diff
@@ -32,8 +32,7 @@ references:
     stigid@ol8: OL08-00-010163
     stigid@rhel8: RHEL-08-010163
 
-platforms:
-    - krb5_server_older_than_1_17-18
+platform: package[krb5-server]<1.17.18
```

Comment:
I have noticed an interesting thing which might be a bug.
Note that here you use the version in the form 1.17.18. But in other cases you use 1.17-18. This is inconsistent and needs to be fixed... but that is not the problem this comment is about.
It seems that there is a problem with generating names of CPE names - in this state it should generate two independent CPE names (one for 1.17-18 and one for 1.17.18). But only one name is generated.
This should not get fixed in this PR probably, but it should get fixed eventually.

Reply:
nice catch! thanks!